test/unique-selector.js

```diff
@@ -35,6 +35,15 @@ describe( 'Unique Selector Tests', () =>
     expect( uniqueSelector ).to.equal( 'body > :nth-child(1)' );
   } );
 
+  it( 'Classes', () =>
```

Comment:
Well, I named the tests after your tests which are not any more descriptive than this one either.